tests/conftest.py

```diff
@@ -135,37 +212,194 @@ def check_socket(address, port):
         s.close()
 
 
+def mirror_contract_data(dest_file_path, src_file_path, src_network, dest_network):
+    dest_data = ""
+
+    with open(src_file_path, "r") as src_file, open(dest_file_path, "r") as dest_file:
+        src_data = json.load(src_file)
+        dest_data = json.load(dest_file)
+        network_data = src_data["networks"][src_network]
+        partial_network_data = {
+            "environment_type": network_data["environment_type"],
+            "indexer_start_block_number": 1,
+            "addresses": network_data["addresses"],
+        }
+        new_network_data = dest_data["networks"][dest_network] | partial_network_data
+        dest_data["networks"][dest_network] = new_network_data
```

Comment:
This could be outside the context manager scope for the with open on files.

Reply:
Then I would need to move the other variables too.